ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,557 @@
+                "callNumber": {
+                  "description": "Call Number identifier assigned to a holding",
+                  "type": "string"
+                }
+              },
+              "required": [
+                "prefix",
+                "suffix",
+                "typeId",
+                "typeName",
+                "callNumber"
+              ]
+            },
+            "shelvingTitle": {
+              "description": "Indicates the shelving form of title",
+              "type": "string"
+            },
+            "acquisitionFormat": {
+              "description": "Format of holdings record acquisition",
+              "type": "string"
+            },
+            "acquisitionMethod": {
+              "description": "Method of holdings record acquisition",
+              "type": "string"
+            },
+            "receiptStatus": {
+              "description": "Receipt status (e.g. pending, awaiting receipt, partially received, fully received, receipt not required, and cancelled)",
+              "type": "string"
+            },
+            "electronicAccess": {
+              "description": "Aggregated electronic access from holdings",
+              "type": "array",
+              "items": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "uri": {
+                      "description": "Electronic access URI",
+                      "type": "string",
+                      "$ref": "../url.json"
+                    },
+                    "name":
+                    {
+                      "description": "Electronic access name",
+                      "type": "string"
+                    },
+                    "linkText": {
+                      "description": "Electronic access link text",
+                      "type": "string"
+                    },
+                    "publicNote": {
+                      "description": "Electronic access public note",
+                      "type": "string"
+                    },
+                    "relationshipId": {
+                      "description": "Electronic access relationship id",
+                      "type": "string",
+                      "$ref": "../uuid.json"
+                    },
+                    "materialsSpecification": {
+                      "description": "Electronic access materials specification",
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "uri",
+                    "name",
+                    "linkText",
+                    "publicNote",
+                    "relationshipId",
+                    "materialsSpecification"
+                  ]
+                }
+              ]
+            },
+            "notes": {
+              "description": "Notes about action, copy, binding etc.",
+              "type": "array",
+              "items": [
+                {
```

Comment:
This fails the build and should be changed to

              "items": {
                "type": "object",
                "properties": {

Reply:
You are right, it looks like I was wrong defined arrays of objects for some entries. I'll fix it, thanks.